pkgs/tools/graphics/goverlay/default.nix

```diff
@@ -1,91 +1,73 @@
 {
-  lib,
-  writeScriptBin,
   bash,
-  stdenv,
+  coreutils,
   fetchFromGitHub,
   fpc,
-  lazarus-qt,
-  wrapQtAppsHook,
-  breeze-qt5,
-  libGL,
-  libGLU,
-  libqt5pas,
-  libX11,
-  coreutils,
   git,
   gnugrep,
+  iproute2,
+  lazarus-qt6,
+  lib,
+  libGL,
+  libGLU,
   libnotify,
+  libqt6pas,
+  libX11,
+  nix-update-script,
   polkit,
   procps,
+  stdenv,
   systemd,
   util-linux,
   vulkan-tools,
   which,
-  nix-update-script,
+  wrapQtAppsHook,
 }:
```

Comment:
For consistency with other packages, please put `lib` as the first attribute in the function call, followed by `stdenv` and `fetchFromGitHub`.

Reply:
The sorting I can split out, I'm not sure how useful the removal of `find-xdg-data-files` is on its own. It was required with 0.7.1 but isn't with 1.2. Unless you're referring of the deletion of `find-xdg-data-files.patch`, than I can easily separate.